beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -1260,40 +1260,6 @@ impl<T: BeaconChainTypes> IntoExecutionPendingBlock<T> for SignatureVerifiedBloc
     }
 }
 
-impl<T: BeaconChainTypes> IntoExecutionPendingBlock<T> for Arc<SignedBeaconBlock<T::EthSpec>> {
```

Comment:
I think this is a good cleanup given the logic mostly duplicates with the `RpcBlock` impl - internally it creates an `RpcBlock` anyway - and we don't really use this much in production code path.